src/datajudge/constraints/stats.py

```diff
@@ -1,6 +1,7 @@
 from typing import Any, Optional, Tuple
 
 import sqlalchemy as sa
+from numpy import typing
```

Comment:
Since this is only needed for typing purposes better wrap it such that doesn't collide with environments not having `numpy`

```python
from typing import TYPE_CHECKING:
    if TYPE_CHECKING:
        from numpy import typing
```